internals/overlord/logstate/fake.go

```diff
@@ -0,0 +1,70 @@
+package logstate
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/canonical/pebble/internals/servicelog"
+)
+
+// Fake sample implementations of logClient
+// TODO: remove this file before merging
```

Comment:
Is this used by the tests?

Reply:
nope it's just for QA. Maybe I'll put in on a separate branch?